apps/studio/schemas/person.ts

```diff
@@ -0,0 +1,39 @@
+import { HeartIcon } from "@sanity/icons";
```

Comment:
See comment above: `UserIcon`

Reply:
Fixed in [`be8cb05`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/be8cb056605ba8708d4ceafed6fa6af9e446787e)